### ISSUE TITLE
Getting KotlinNullPointerException

### ISSUE THREAD

**Issue by SMUsamaShah** [zielu/GitToolBox#271]
GitToolBox fails to start and gives the following stacktrace.

IntelliJ Ultimate 2020.2.3
GitToolBox 202.2.0

```
com.google.common.util.concurrent.UncheckedExecutionException: kotlin.KotlinNullPointerException
	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2051)
	at com.google.common.cache.LocalCache.get(LocalCache.java:3951)
	at com.google.common.cache.LocalCache.getOrLoad(LocalCache.java:3974)
	at com.google.common.cache.LocalCache$LocalLoadingCache.get(LocalCache.java:4958)
	at zielu.intellij.guava.GuavaExtensionsKt.getSafe(GuavaExtensions.kt:9)
	at zielu.gittoolbox.blame.calculator.CachingBlameCalculator.annotate(CachingBlameCalculator.kt:44)
	at zielu.gittoolbox.blame.BlameLoaderImpl.annotate(BlameLoaderImpl.java:27)
	at zielu.gittoolbox.blame.BlameCacheImpl$AnnotationLoader.load(BlameCacheImpl.java:226)
	at com.codahale.metrics.Timer.timeSupplier(Timer.java:136)
	at zielu.gittoolbox.blame.BlameCacheImpl$AnnotationLoader.run(BlameCacheImpl.java:212)
	at zielu.gittoolbox.blame.BlameCacheExecutor$1.run(BlameCacheExecutor.java:64)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:935)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressAsync$5(CoreProgressManager.java:442)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$3(ProgressRunner.java:235)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:170)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:629)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:581)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:157)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:235)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1700)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: kotlin.KotlinNullPointerException
	at zielu.gittoolbox.blame.calculator.persistence.BlameCalculationPersistence.filePath(BlameCalculationPersistence.kt:46)
	at zielu.gittoolbox.blame.calculator.persistence.BlameCalculationPersistence.getBlame(BlameCalculationPersistence.kt:50)
	at zielu.gittoolbox.blame.calculator.CachingBlameCalculatorLocalGateway$loadFromPersistence$1.get(CachingBlameCalculatorLocalGateway.kt:29)
	at zielu.gittoolbox.blame.calculator.CachingBlameCalculatorLocalGateway$loadFromPersistence$1.get(CachingBlameCalculatorLocalGateway.kt:13)
	at com.codahale.metrics.Timer.timeSupplier(Timer.java:136)
	at zielu.gittoolbox.blame.calculator.CachingBlameCalculatorLocalGateway.loadFromPersistence(CachingBlameCalculatorLocalGateway.kt:28)
	at zielu.gittoolbox.blame.calculator.CachingBlameCalculator.getFromPersistence(CachingBlameCalculator.kt:72)
	at zielu.gittoolbox.blame.calculator.CachingBlameCalculator.calculate(CachingBlameCalculator.kt:62)
	at zielu.gittoolbox.blame.calculator.CachingBlameCalculator.access$calculate(CachingBlameCalculator.kt:14)
	at zielu.gittoolbox.blame.calculator.CachingBlameCalculator$loader$1.loadInContext(CachingBlameCalculator.kt:18)
	at zielu.gittoolbox.blame.calculator.CachingBlameCalculator$loader$1.loadInContext(CachingBlameCalculator.kt:16)
	at zielu.intellij.guava.ThreadLocalCacheLoader.load(ThreadLocalCacheLoader.kt:19)
	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3529)
	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2278)
	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2155)
	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2045)
	... 27 more
```

**Comment by zielu** [comment]
In the mean time you can disable blame persistence:
1.  Use Find Action... to seach for `Registry...`
2. Find key `zielu.gittoolbox.blame.cache.persistent`
3. Uncheck it and reopen project

**Comment by zielu** [comment]
Btw is our git repository root outside of project directory ?

**Comment by zielu** [comment]
Fixed in 202.2.1

**Comment by SMUsamaShah** [comment]
> Btw is our git repository root outside of project directory ?
Yes it is. 

Thank you very much for fixing this. Was feeling short sighted without it.
